Packages/com.unity.inputsystem/InputSystem/Plugins/EnhancedTouch/Touch.cs

```diff
@@ -42,212 +60,245 @@ public struct Touch : IEquatable<Touch>
         /// <summary>
         /// Whether this touch record holds valid data.
         /// </summary>
-        /// <value>If true, the data contained in the touch is valid.</value>
         /// <remarks>
+        /// <para>
         /// Touch data is stored in unmanaged memory as a circular input buffer. This means that when
         /// the buffer runs out of capacity, older touch entries will get reused. When this happens,
         /// existing <c>Touch</c> instances referring to the record become invalid.
-        ///
+        /// </para>
```

Comment:
I have gotten xmldoc error any time I mix non-para sections with para sections hence why I have used them since the error message say do not mix para with inline content. Maybe there is something I do not understand about them?